### PR TITLE
Update PullToRefresh specs

### PR DESCRIPTION
**Pull request by sergigracia**
#### :tophat: What? Why?

Update the pull to refresh specs to follow the new design 2.0.
#### :dart: What should QA test?
1. Open the wall and scroll up & down. 
2. Do a pull to refresh.
3. Expected: the loader should appear and disappear automatically.
4. Expected: the icon should be the new one.
#### :clipboard: Developer checklist
- [x] Unit Tests (if necessary)
- [x] UI Snapshot Tests (if necessary)
- [ ] Added accessibility tags to any interactive element
- [x] Added reporting for GA (events + screen views) / Errors / Exceptions
- [ ] Documentation

### CODE REVIEW

File: KoaPullToRefresh/KoaPullToRefresh.h [hunk 1]
```diff
@@ -40,7 +40,7 @@ typedef NSUInteger KoaPullToRefreshState;
 @property (nonatomic, strong) UIColor *textColor;
 @property (nonatomic, strong) UIFont *textFont;
 @property (nonatomic, strong, readonly) UILabel *titleLabel;
-@property (nonatomic, strong, readonly) UILabel *loaderLabel;
+@property (nonatomic, strong, readonly) UILabel *loaderControl;
```
Comment (Lascorbe) [inline KoaPullToRefresh/KoaPullToRefresh.h]: Creo que seria mejor `loadControl`

File: KoaPullToRefresh/KoaPullToRefresh.m [hunk 2]
```diff
@@ -124,6 +127,8 @@ - (void)setShowsPullToRefresh:(BOOL)showsPullToRefresh {
     }
 }
 
+#pragma mark - Hello folks !
```
Comment (juolgon) [inline KoaPullToRefresh/KoaPullToRefresh.m]: Delete this line please

File: KoaPullToRefresh/KoaPullToRefresh.m [hunk 3]
```diff
@@ -216,17 +221,11 @@ - (void)layoutSubviews
...
-}
-
 - (void)setScrollViewContentInsetForLoading {
     UIEdgeInsets currentInsets = self.scrollView.contentInset;
-    //CGFloat offset = MAX(self.scrollView.contentOffset.y * -1, 0);
-    //currentInsets.top = MIN(offset, self.originalTopInset + self.bounds.size.height);
-    currentInsets.top = self.originalTopInset + self.bounds.size.height;
+    CGFloat offset = MAX(self.scrollView.contentOffset.y * -1, 0);
+    currentInsets.top = MIN(offset, self.originalTopInset + self.bounds.size.height);
+    //currentInsets.top = self.originalTopInset + self.bounds.size.height;
```
Comment (Lascorbe) [inline KoaPullToRefresh/KoaPullToRefresh.m]: **Mucho** wapo to' ahi

Also, normal emojis -> :beers:

**Comment by Lascorbe** [comment]
:+1:

**Comment by juolgon** [comment]
:+1:

**Pull request closed without merging**